### DISCUSSION TITLE
Could have two different online docs sites/repos: public, private?

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_docs#13] [Announcements]
We discussed this briefly a while back: basically internal vs external pages. Maybe there is also a way to add internal webpages (that require some sort of password?) to the same repo? Not urgent for now.

**Comment by glopesdev** [comment]
@jkbhagatio GitHub repos on paid org accounts allow you to have privately published sites that share the GitHub authentication policies: https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/changing-the-visibility-of-your-github-pages-site

Assuming the two websites have non-overlapping information I think this would probably be the easiest low-friction way to have two docs repos.
